app/api/base.py

```diff
@@ -34,6 +41,10 @@ def add_user_resources(self):
             'coin': {
                 'amount': self.search_for_resources(),
                 'picture': url_for ('static', filename=f'images/coin.png')
+            },
+            'diamond': {
+                'amount': self.search_for_diamond(),
+                'picture': url_for ('static', filename=f'images/diamond.png')
```

Comment:
Why are these f strings?
```suggestion
                'picture': url_for ('static', filename='images/diamond.png')
```

Reply:
my bad, copy pase :(

Reply:
You don't need it in the other ones either

Reply:
Yea i saw, thanks